pages/1.11/installing/ent/custom/system-requirements/index.md

```diff
@@ -49,8 +47,6 @@ Here are the agent node hardware requirements.
 
```

Comment:
Add the following text, before the table "For supported Operating Systems and Environments are listed on https://docs.mesosphere.com/version-policy/"